packages/server/postgres/migrations/1625124096757_addUserEmailUniqueConstraint.ts

```diff
@@ -0,0 +1,20 @@
+import {MigrationBuilder, ColumnDefinitions} from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createIndex('User', ['email'], {
+    name: 'User_email_index',
+    unique: true
+  })
```

Comment:
I wasn't sure if this pgm helper function creates indexes concurrently by default, and it looks like it doesn't. But you can pass it in as an option! can you add that? doing it will prevent the table from locking while the index is being created.